akka-cluster/src/main/scala/akka/cluster/ClusterRemoteWatcher.scala

```diff
@@ -73,8 +71,6 @@ private[cluster] class ClusterRemoteWatcher(
   override def receive = receiveClusterEvent orElse super.receive
 
   def receiveClusterEvent: Actor.Receive = {
-    case WatchRemote(watchee, watcher) if clusterNodes(watchee.path.address) ⇒
```

Comment:
`WatchRemote` message must be handled by the remote watcher for updating its watching map. This is necessary because remote watcher has now the responsibility to propagate the `DeathWatchNotifcation` to all watchers.

The heartbeat process will be bypassed for cluster members thanks to the override of the `watchNode` method